cmd/version/list.go

```diff
@@ -174,7 +180,7 @@ var listCmd = &cobra.Command{
 		}
 
 		// Format output.
-		switch strings.ToLower(listFormat) {
+		switch normalizedFormat {
 		case "text":
 			return formatReleaseListText(releases)
```

Comment:
Can we rename text to pretty or table. Text is not a great representation. What do we do in our other list commands?